robottelo/helpers.py

```diff
@@ -253,3 +257,54 @@ def remove_katello_ca(hostname=None):
             )
             if result.return_code != 0:
                 raise AssertionError('Failed to reset the rhsm.conf')
+
+
+def get_available_capsule_port():
```

Comment:
This is relying on socat being available and that the testing server have some predefined infrastructure. With that said, I think a new configuration should be added to the robottelo.properties so we can configure the expected port range as well as being able to skip tests that rely on extra infrastructure.

Also would be awesome have some documentation on this function explaining all the commands that it will use on the server and also the expected infrastructure.